src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,114 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  items: string[],
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (number: number) => void,
+  from: number,
+  to:number,
+};
+
+export const Pagination: React.FC<Props> = ({
+  items,
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+  from,
+  to,
+}) => {
+  const pagesCount = Math.ceil(total / perPage);
+  const pageItems = [];
+
+  const onClickHandle = (item: number) => {
+    onPageChange(item);
+  };
+
+  //eslint-disable-next-line no-plusplus
```

Comment:
use map instead of for loop

Reply:
i don't understand how I can use map, cause there writes that I can't use map with numbers...